### DISCUSSION TITLE
请教大佬：所有的notes的背景突然变成深绿色

### DISCUSSION THREAD

**Discussion by doczhengexplorealone** [windingwind/zotero-better-notes#853] [General]
不知是插件冲突还是其他原因，进行zotcard设置后，发现所有的原有的notes的背景突然变成深绿色，新建的note也是，删除zotcard，重新安装zotero-better-notes之后，新建note仍然为深绿色，求教如何改为之前的默认背景？
![Uploading 微信截图_20240110203619.png…]()

**Comment by l0o0** [comment]
你的图片，是不是没有上传成功？@doczhengexplorealone

**Comment by doczhengexplorealone** [comment]
@l0o0 是的不好意思没注意到，已在zotero论坛上解决，过程如下：
bug情况截屏：https://1drv.ms/i/s!Aq5EffuZvmRwiDXmDUMvcpaBPQkU?e=IJL82Q，
troubleshooting：
@dstillman回复:in fact, it looks like ZotCard does this:
[https://github.com/search?q=repo:018/zotcard+note.css&amp;type=code](https://github.com/search?q=repo%3A018%2Fzotcard+note.css&type=code)
It really should not be doing this, so we'll contact the plugin developer.
解决办法：If you go to the Config Editor in the Advanced tab of the Zotero settings and search for extensions.zotero.note.css, is it set to a value? If so, reset it. (首选项-高级设置-编辑器-搜索框检索“extensions.zotero.note.css”-删除extensions.zotero.note.css之后的内容(即恢复默认状态))

**Comment by l0o0** [comment]
正常情况下，删除插件后，修改过的配置，不会恢复。如果插件修改的话，确实很难发现呢

**Reply by doczhengexplorealone** [reply]
是的，所以这真是为难到我了，还好最终通过向大家提问的方式解决了
